django/db/models/query.py

```diff
@@ -1423,6 +1423,14 @@ def values_list(self, *fields, flat=False, named=False):
                     "field."
                 )
             elif not fields:
+                # RemovedInDjango70Warning: When the deprecation ends, raise
+                # TypeError instead.
```

Comment:
```suggestion
                # RemovedInDjango70Warning: When the deprecation ends, replace with:
                # raise TypeError(
                #     "'flat' is not valid when values_list is called with no fields."
                # )
```
Given other `TypeError`s here have a specific message, I think it makes sense to define one here